src/tasks/DockerPseudoterminal.ts

```diff
@@ -114,3 +139,21 @@ export class DockerPseudoterminal implements Pseudoterminal {
         this.writeEmitter.fire(`\x1b[${color}${message}\x1b[0m`);
     }
 }
+
+/**
+ * Loosely based on https://github.com/microsoft/node-pty/blob/master/typings/node-pty.d.ts, but downloading that is difficult and actually installing
+ * the node-pty package is even more difficult (lots of native code)
+ */
+interface nodepty {

```

Comment:
Any reason not to keep this in a `src\types\node-pty.d.ts` file?

Reply:
Not particularly, was just thinking this was probably the only place it would be needed. But I don't feel strongly at all.